docs/executing/cloud.rst

```diff
@@ -454,3 +454,106 @@ the `FUNNEL_SERVER_USER` and  `FUNNEL_SERVER_PASSWORD` AS environmental variable
     $ export FUNNEL_SERVER_USER=funnel
     $ export FUNNEL_SERVER_PASSWORD=abc123
 
+-----------------------------------------------------------------
+Executing a Snakemake workflow via Azure Batch
+-----------------------------------------------------------------
+
+First, install the `Azure CLI <https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest>`_.
+Then install Azure related dependencies:
+
+.. code:: console
+
+    conda create -c bioconda -c conda-forge  msrest azure-batch azure-storage-blob azure-mgmt-batch azure-identity
+
+
+Data in Azure Storage
+~~~~~~~~~~~~~~~~~~~~~~
+
+Using this executor typically requires you to start with large data files
+already in Azure Storage, and then interact with them via Azure Batch. An easy way to do this is to use the
+`azcopy <https://docs.microsoft.com/en-us/azure/storage/common/storage-use-azcopy-v10>`__.
+command line client. For example, here is how we might upload a file
+to storage using it:
+
+.. code-block:: console
+
+    $ azcopy copy mydata.txt "az://snakemake-bucket/1/mydata.txt"
```

Comment:
how does this "az://" protocol work ?  I have not ever seen that nor can I find evidence of it in the azcopy docs ?

Reply:
sorry this should be the other way

azcopy copy mydata.txt "https://$account.blob.core.windows.net/snakemake-bucket/1/mydata.txt"

Reply:
No i wish. I will try to work on it for azcopy.